.github/workflows/pytest.yaml

```diff
@@ -144,6 +143,6 @@ jobs:
       run: make --directory=doc html
 
     - name: Upload test coverage to Codecov.io
-      uses: codecov/codecov-action@v1.2.1
+      uses: codecov/codecov-action@v2
```

Comment:
@LauWien the discrepancy might also be due to this.

Compare
- https://app.codecov.io/gh/iiasa/message_ix/branch/main at the bottom, the code is in the directory "message_ix"
- https://app.codecov.io/gh/iiasa/message_ix/branch/feature%2Fsolve-args-config at the bottom, we instead see "message_ix/message_ix".

The difference of 47 lines corresponds to `.testing.nightly`, so I think this might resolve itself at the next nightly test run.

Reply:
Ah, yes, I'll then merge!